org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -16,10 +30,10 @@ public static void playSound(double[] sound) {
    /**
     * Plays the sound referenced by the file name.
     * 
-    * @param filename an array of samples to play.
-    * @throws SoundException if the file can't be found in the project.
+    * @param filename the file to play in the asset manager.
+    * @throws FileNotFoundException if the file can't be found in the project.
     */
-   public static void playSound(String filename) throws SoundException {
+   public static void playSound(String filename) throws FileNotFoundException {
```

Comment:
Just to double check. Students will have learned about exception handling at this point? i.e. that was a concern when we were building the Neighborhood API - students wouldn't know how to handle exceptions yet.

Reply:
Jamila indicated that using exceptions is fine as they can work around them with helper code if they need to.